test/Microsoft.ServiceFabric.Services.Remoting.Tests/ExceptionDeserializerTest.cs

```diff
@@ -85,7 +86,8 @@ public class ConvertRemoteException : ExceptionDeserializerTest
             public void ReturnsOriginalExceptionIfItIsKnownExceptionType()
             {
                 // Arrange
-                var exceptionSerializer = Remoting.V2.Runtime.ExceptionSerializer.CreateDefault(Enumerable.Empty<Remoting.V2.Runtime.IExceptionConvertor>(), null);
+                var exceptionSerializer = Remoting.V2.Runtime.ExceptionSerializer
+                        .CreateDefault(Enumerable.Empty<Remoting.V2.Runtime.IExceptionConvertor>(), new FabricTransportRemotingListenerSettings());
```

Comment:
I think it's OK to use the FabricTransportRemotingListenerSettings here, but since we have an interface, it would be best to use Mock<IExceptionSerializerSettings> and test the ExceptionDeserializer without the dependendcy.